core/src/epicli/data/common/ansible/playbooks/roles/postgresql/molecule/debian-repmgr/prepare.yml

```
@@ -0,0 +1,6 @@
+---
+- name: Converge
```

```suggestion
- name: Prepare
```
Shouldn't be `Prepare`?